examples/render/contour-deployment.yaml

```diff
@@ -4908,7 +4908,7 @@ spec:
       containers:
       - name: contour
         image: ghcr.io/projectcontour/contour:main
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
This was set this way on the `main` branch because if not you'd never get an updated version of it. The versioned examples do change this to `Always`.

Reply:
@stevesloka I think you're suggesting that we move [this logic](https://github.com/projectcontour/contour/blob/main/hack/generate-deployment.sh#L56-L66) out of this script that generates the rendered manifests, and into https://github.com/projectcontour/contour/blob/main/hack/release/make-release-tag.sh so it only applies to versioned manifests?

Reply:
I guess my point was on the `main` branch, if you used this code (which you really shouldn't but only in a dev/test env), then the 2nd time you updated you wouldn't get the new version of Contour since there's a version already present on the machine. =)

Reply:
Yep makes sense -- this has been true for the contour serve, bootstrap and shutdown manager image refs for a long time so odds are nobody's actually using this manifest anyway, but filed https://github.com/projectcontour/contour/issues/4404 to track the additional change